.github/workflows/test.yml

```diff
@@ -160,7 +150,7 @@ jobs:
       - name: Install dependencies
         shell: bash -l {0}
         run: |
-          mamba install python=3.8 python-blosc lz4 --file conda.txt -c conda-forge
+          mamba install python-blosc lz4 -c conda-forge
```

Comment:
Not sure if we would want to pin `python-blosc` and `lz4` and include them in the `dev` environments

Reply:
Please forgive my ignorance, I am also not aware of these packages, probably dependency packages for some other packages, Not sure though! 🤔